web/vtctld2/package.json

```diff
@@ -53,6 +53,6 @@
     "protractor": "3.3.0",
     "ts-node": "1.2.1",
     "tslint": "3.13.0",
-    "typescript": "^2.0.0"
+    "typescript": "=2.0.10"
```

Comment:
I had to pin down the `typescript` version to be able to bundle the assets using `make web_build`. However I'm not sure if this is only needed in MacOS or something else environment specific.